README.md

```diff
@@ -650,6 +650,7 @@ Table of Contents
   * [Upstash Kafka](https://upstash.com/kafka) - Serverless Kafka Cloud offering with per-request-pricing. It has a free tier with max 10000 messages per day.
   * [webpushr](https://www.webpushr.com/) - Web Push Notifications - Free for upto 10k subscribers, unlimited push notifications, in-browser messaging
   * [Scramjet Cloud Platform Beta](https://www.scramjet.org/#join-beta) - An end-to-end stream processing platform in free beta and offering 15 petabyte-seconds of free compute after the beta ends.
+  * [Raven](https://www.raven.dev/) - API to manage notifications across SMS, Email, Push, Slack, WA & more, First 10,000 events free every month
```

Comment:
raven.dev seems to have a weird signup page, submitting details and waiting for them to email back.

Maybe we can have the signup url instead of a form to signup?

these were the emails i received,
1/2

Hi REDACTED - Thank you for requesting access. Excited to see how we can be of help.

I would love to have a brief [15min chat](https://calendly.com/priyanshs/15-minute-meeting) to understand the use-case. We can even help you set up in a few minutes.

In case you want the platform access only, I'll share the links. Let me know what works best.
-- 
Best,

Priyansh Srivastava
Co-Founder, [Raven](http://www.raven.dev/)

2/2
Sure REDACTED, here you go -

Console & onboarding steps

Here is our console link where you can sign-up directly: redacted(not a dynamic url[no tokens/code for signup])t
Steps for onboarding:
1. Setup Integrations: Go to Integrations tab and add a provider of your own for any channel (can use Raven SMS, Raven Email provider for testing)
2. Add Event: Create a new event > Select the channels you want to send notifications to and the providers for those channels > Add template > Save Event
3. Fire Event: Use the [Send event API](https://docs.ravenapp.dev/api-reference/send-event) to fire the event that was created earlier. All you need to pass is the event name, user details and any dynamic params.
4. Logs & Metrics: You will be able to see the fired event in the logs section and the response obtained from the provider. For aggregated metrics like delivery rates, failure reasons etc, you can visit the metrics tab.

Feel free to reach out anytime if you have any questions!

Reply:
Yeah this is not acceptible for the list

Reply:
ok, i remove raven.dev